### PR TITLE
Add some OSS docs

### LINKED ISSUE #10: Create OSS supporting docs
I feel as of now the project lacks a couple of documents to maintain the open-source spirit. I plan on adding:
- Add CODEOWNERS
- A Code Of Conduct
- Create a Contributing guide
- Create LICENSE
- Create Issue Templates
- Create PR Templates

### PR DESCRIPTION
**Pull request by Rishit-dagli**
I plan on adding some docs to this repository to maintain the open-source spirit.

---

Closes #10

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```
Comment (ginomessmer) [inline CONTRIBUTING.md:17]: Should be renamed to main.

```suggestion
1. Fork the repo and create your branch from `main`.
```

Comment (ginomessmer) [inline CONTRIBUTING.md:17]: Should be renamed to main.

```suggestion
1. Fork the repo and create your branch from `main`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```
Comment (ginomessmer) [inline CONTRIBUTING.md:21]: Even though the current code linter checks for accessibility issues, should we consider adding a special note about A11Y?

**Review by ginomessmer — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Rishit-dagli) [reply CONTRIBUTING.md:21]: I think accessibility linting should be covered under point 5, but we could add it as a separate bullet too?

Comment (ginomessmer) [reply CONTRIBUTING.md:21]: I think you're right. Though I've been thinking about e.g. [Accessibility Insights](https://accessibilityinsights.io/) but we can also consider it at a later date. What do you think?

**Review by ginomessmer — APPROVED** [review]

**Pull request merged**